src/main/resources/templates/macros/json/hooks.vm

```diff
@@ -1,5 +1,9 @@
 #macro(includeHooks $keyword, $hooks, $status, $level)
 
+#if($hide_empty_hooks)
+  #set($hooks = $util.eliminateEmptyHooks($hooks))
+#end
+
 #if(!$hooks.isEmpty())
```

Comment:
See type-change comment above at Util.java: 
If NOT $hide_empty_hooks, then $hooks still is of type array, isn't it? And array does not have method .isEmpty(), I think?
Or is there some kind of runtime conversion?
Not sure if this is a problem.

Reply:
no flag -> stays unchanged as it is right now

isEmpty is the method of velocity template, not pure java in this case